format/FlightSql.proto

```diff
@@ -867,6 +867,167 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+enum SqlDataType {
+  SQL_TYPE_UNKNOWN_TYPE = 0;
+  SQL_TYPE_CHAR = 1;
+  SQL_TYPE_NUMERIC = 2;
+  SQL_TYPE_DECIMAL = 3;
+  SQL_TYPE_INTEGER = 4;
+  SQL_TYPE_SMALLINT = 5;
+  SQL_TYPE_FLOAT = 6;
+  SQL_TYPE_REAL = 7;
+  SQL_TYPE_DOUBLE = 8;
+  SQL_TYPE_DATETIME = 9;
+  SQL_TYPE_INTERVAL = 10;
+  SQL_TYPE_VARCHAR = 12;
+}
+
+enum SqlDatetimeSubcode {
+  option allow_alias = true;
```

Comment:
As I said this implementation was planned to be used by JDBC and ODBC, and the values set in the enum should correspond to the same used by them. In some cases, some of the valeus are the same, like:
`SQL_SUBCODE_TIME = 2;
  SQL_SUBCODE_MONTH = 2;`

So the alias option was used to set different constant to the same value.

Another approach would be to set a generic name and then with a comment specify that values also refers to other  constants.

Let me know which you prefer

Reply:
As I understand it, the Flight SQL spec is not user-facing but intended for implementors of bridges/drivers/adapters. So having convenience aliases for the same numeric values doesn't seem very useful?
(unless it can simplify the implementation of such bridges/drivers/adaptaters?)